tests/controls/windowtest.cpp

```diff
@@ -144,11 +194,36 @@ TEST_CASE_METHOD(WindowTestCase, "Window::FocusEvent", "[window]")
 
 TEST_CASE_METHOD(WindowTestCase, "Window::Mouse", "[window]")
 {
+    wxInitAllImageHandlers();
     wxCursor cursor(wxCURSOR_CHAR);
     m_window->SetCursor(cursor);
 
     CHECK(m_window->GetCursor().IsOk());
 
+#if defined( __WXMSW__ ) || defined( __WXOSX__ )
+    wxCursor resCursor1( "horse" );
+    CHECK( resCursor1.IsOk() );
+#endif
+#ifdef __WXGTK__
+    // According to documentation cursors on X/GTK are different than in Windows/OSX.
+    // On X/GTK, using the constructor with wxString parameter is loading the cursor
+    // from the external file and not creating it from embedded bits/resources
+    // Therefore different path are used for creating embedded cursors
+    wxCursor resCursor1( (const char *) horse, 32, 32, 16, 0, nullptr, wxWHITE, wxBLACK );
+    CHECK( resCursor1.IsOk() );
+#endif
+#ifdef __WXOSX__
+    wxRemoveFile( wxStandardPaths::Get().GetResourcesDir() + "/horse.png" );
```

Comment:
It seems like a very bad idea to remove images from the .app resources. Why not try to load something that doesn't exist, like `horse2`?

Reply:
@MaartenBent ,
For OSX,  there are 2 possibilities png and cur. So if png doesn't exist I try cur.
Since most of the time people develop on Windows and then just compile and test the code on other platforms, I figured it will be better to test both scenarios.

Or maybe you have an idea on how to do that?

Reply:
Like I said, use a filename that does not exist. Or a filename that only has a .cur. Something like
`cp -f $(WXTOPDIR)tests/horse.cur $(BUNDLE)/Resources/horse2.cur` and then you try to load `horse2` which doesn't have a png but only cur.